sdk/monitor/azure-monitor-query/src/main/java/module-info.java

```diff
@@ -9,9 +9,9 @@
     requires transitive com.azure.core.experimental;
     exports com.azure.monitor.query;
     exports com.azure.monitor.query.models;
-    opens com.azure.monitor.query.log.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.monitor.query.metrics.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.monitor.query.metricsdefinitions.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.monitor.query.metricsnamespaces.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.monitor.query.models to com.fasterxml.jackson.databind, com.azure.core;
+    opens com.azure.monitor.query.implementation.logs.models to com.fasterxml.jackson.databind, com.azure.core;
```

Comment:
>.implementation.logs [](http://example.com/codeflow?start=33&length=20)

why the change in the namespace?

Reply:
Jonathan suggested that we have `implementation` as the root and then fork into logs, metrics etc. This follows the pattern we've used in other places.